mulog-cloudwatch/src/com/brunobonacci/mulog/publishers/cloudwatch.clj

```diff
@@ -0,0 +1,106 @@
+(ns com.brunobonacci.mulog.publishers.cloudwatch
+  (:require [com.brunobonacci.mulog.publisher :as p]
+            [com.brunobonacci.mulog.buffer :as rb]
+            [com.brunobonacci.mulog.utils :as ut]
+            [cognitect.aws.client.api :as aws]
+            [cheshire.core :as json]
+            [cheshire.generate :as gen]))
+
+
+(defn- has-failures?
+  [rs]
+  (not (= {} rs)))
+
+
+(defn- create-cloudwatch-client
+  [params]
+  (aws/client params))
+
+
+(defn- publish!
+  [cloudwatch-client group-name stream-name records]
+  (let [create-rs (aws/invoke cloudwatch-client {:op :CreateLogStream
+                                                 :request {:logGroupName group-name
+                                                           :logStreamName stream-name}})]
```

Comment:
here you create the log stream on every publish! invocation!!!
I think the creation of the log stream should happen only once at the start/initialization.